### ISSUE TITLE
[Bug]: Can't shim 'bin\mtn.exe': File doesn't exist in mtn 3.5.0

### ISSUE THREAD

**Issue by reaseno** [ScoopInstaller/Main#5578]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

mtn

### Expected/Current Behaviour

Installing 'mtn' (3.5.0) [64bit] from main bucket
Loading mtn-3.5.0-win64.zip from cache
Checking hash of mtn-3.5.0-win64.zip ... ok.
Extracting mtn-3.5.0-win64.zip ... done.
Linking ~\scoop\apps\mtn\current => ~\scoop\apps\mtn\3.5.0
Creating shim for 'mtn'.
Can't shim 'bin\mtn.exe': File doesn't exist.

### Steps to Reproduce

```console
scoop install mtn
```

### Possible Solution

Change the path where the mtn.exe is searched. It's now in package root.

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
473b6356e (HEAD -> master, origin/master, origin/HEAD) wavebox: Update to version 10.122.30.2

'main' bucket:
d619dcb3f (HEAD -> master, origin/master, origin/HEAD) terragrunt: Update to version 0.55.12

'naderi' bucket:
f4c61f18 (HEAD -> master, origin/master, origin/HEAD) wavebox-portable: Update to version 10.122.30.2

'nerd-fonts' bucket:
7ee8f1b (HEAD -> master, origin/master, origin/HEAD) GlowSansJ-Condensed: Update to version 0.93
```

### Scoop Config

```console
last_update                       scoop_branch scoop_repo
-----------                       ------------ ----------
2024-03-06T12:05:23.3571204+01:00 master       https://github.com/ScoopInstaller/Scoop
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      5.1.19041.4046
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.4046
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by reaseno** [comment]
What is still missing for publication?

**Comment by aliesbelik** [comment]
Fixed by #5542.

**Comment by aliesbelik** [comment]
Fixed, could be closed.
